manifests/r/RStudio/quarto/1.1.165/RStudio.quarto.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.1.165
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+Commands:
+- quarto
+AppsAndFeaturesEntries:
+- DisplayName: Quarto
+  Publisher: Rstudio, PBC
+  ProductCode: '{F72386FC-CBA0-4DE6-A92A-7D0CC6E34625}'
+  InstallerType: wix
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  Scope: user
+  InstallerUrl: https://github.com/quarto-dev/quarto-cli/releases/download/v1.1.165/quarto-1.1.165-win.msi
+  InstallerSha256: 423D31C68D47388E2A4656BF5B964798E94E743B0FDD2647376EB0D2EC245A72
+  InstallerSwitches:
+    Custom: ALLUSERS=2 MSIINSTALLPERUSER=1

```

Comment:
I borrows this from the [pandoc manifest](https://github.com/microsoft/winget-pkgs/blob/3623cc6e0fe3d56c9dce5d729f2198fdc1aceffc/manifests/j/JohnMacFarlane/Pandoc/2.19.2/JohnMacFarlane.Pandoc.installer.yaml#L20-L21), except that `ALLUSERS=2` is not sufficient for quarto. You're right that the msi installer can determine to some degree, whether to install it with a user and machine scope (i.e. in a non-elevated context it might install it with a user scope - I haven't tested this). 
That said, I did not manage to install quarto with user scope without these switches.
Maybe @OfficialEsco (who [showed me](https://github.com/microsoft/winget-pkgs/pull/18505#issuecomment-870709267) the "two nodes with different scopes from the same installer" trick) or @vedantmgoyal2009 (who built the pandoc manifests AFAIK) know more?

Reply:
The InstallerType provides the Silent, SilentWithProgress, Log and Install Location switch, any other like scope and reboot needs to be included in the Custom switches.

Reply:
I am wondering because usual switches for this type of installer are 

- Silence mode without interaction: `/quiet`
- SilentWithProgress: `/passive`

There are some short version also I think like `/qn` for not interface 

Details can be seen by displaying the help on the package:
````
.\quarto-1.1.165-win.msi -?
````
This will open a box. Example for French version:
![image](https://user-images.githubusercontent.com/6791940/187693735-b81094e3-81cc-4f31-8d56-8157de54356e.png)

But it seems indeed no explicit switch for the Machine vs User install. I am not even sure winget supports them all.

So it seems official doc for PROPERTY VALUES than `winget` support is here: https://docs.microsoft.com/en-us/windows/win32/msi/property-reference

* ALLUSER: https://docs.microsoft.com/en-us/windows/win32/msi/allusers
  Value 2 seems to be a specific one, which can indeed allow `MSIINSTALLPERUSER`. It seems important to use `ALLUSER=2` with `MSIINSTALLPERUSER` instead of `ALLUSER=""` for old windows. 
* MSIINSTALLPERUSER: https://docs.microsoft.com/en-us/windows/win32/msi/msiinstallperuser
  This seems important for old version. 
  
 It seems like default should be per user thought, I will check which defaults we are setting on our MSI packaging.

Reply:
Does this need any changes from your point of view @cderv?

Reply:
I have checked our installer and we respect `ALLUSERS="" OR ALLUSERS = 2` for User scoped installation. 
From doc above using ALLUSERS = 2 seems to require MSIINSTALLPERUSER. 

Maybe using `ALLUSERS=""` here would be better but I really don't know the full difference. 
So I would keep it like that as it seems to work ok. 

Thank you @dpprdan

Reply:
I tried `ALLUSERS=""` (as in `msiexec /passive /i quarto-1.1.163-win.msi ALLUSERS=""`) and it installed it with machine scope (i.e. in `C:/Program Files/...`). But yeah, I guess it _should_ work, so don't know why it is not.

Reply:
Ok good to know. thanks a lot !